packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -134,7 +135,8 @@ export function HiddenSelect<T>(props: HiddenSelectProps<T>) {
         autoComplete={selectProps.autoComplete}
         name={name}
         disabled={isDisabled}
-        value={state.selectedKey ?? ''} />
+        value={state.selectedKey ?? ''}
+        data-rsp-a11y-ignore />
```

Comment:
Looks like we only need this on the container h/t @LFDanLu 

```suggestion
        value={state.selectedKey ?? ''} />
```